bootstrap/src/main/java/com/splunk/opentelemetry/javaagent/bootstrap/GlobalMetricsTags.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.javaagent.bootstrap;
+
+import io.micrometer.core.instrument.Tag;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class holds the list of tags that should be added to all meters registered by the javaagent.
+ * It needs to be loaded by the bootstrap classloader to be accessible inside instrumentations.
+ */
+public final class GlobalMetricsTags {
```

Comment:
I don't want this thought to get in the way of this PR...but I would love for us to also consider ways of trying to avoid this static/global pattern down the road...even if it means passing the tags into all instrumentations that need it.  Or something.  I dunno....I just worry about this being hard to maintain.

Reply:
I also realize that passing it into an AutoService may be difficult/impossible...but still.  😁

Reply:
I first looked into setting global tags in the meter registry itself, which would be a much cleaner solution -- no need to remember about passing global tags to your new meter, the registry would it automagically. Unfortunately it required digging really deep into `MeterRegistry` internals and overriding >10 different methods to make this work 🥶  

And the fact that those tags need to be accessible both in the agent and application classloaders makes "fancy" (non-global variable 😂 ) implementation much harder...

Reply:
Yeah, of course.  Seems reasonable for now then.